src/game/Player.cpp

```diff
@@ -2447,14 +2447,12 @@ bool Player::IsGroupVisiblefor (Player* p) const
 
 bool Player::IsInSameGroupWith(Player const* p) const

```

Comment:
Changed it back and added config support, now working with Visibility.GroupMode = 1.